### ISSUE TITLE
KPI Donut chart can't switch background off

### ISSUE THREAD

**Issue by pieter248** [bliu5379/PayPal-PowerBI-Custom-Visuals#9]
Hi, I am switching the background off for this visual so that the chart underneath it can be visible but the background remains and therefore it's blocking the chart behind it. How can I turn the background off effectively? I need to create 2 donut kpi guages inside each other

**Comment by cjsturges** [comment]
I'd like to be able to turn the background off as well.  It's a really nice visual, but only if you can make the background transparent.

**Comment by Kev2222** [comment]
Agreed - our UI guys would like us to have a donut within a donut but can't do this with the background of the circle - although not the 'background' being a solid colour.  I'm afraid I don't have the skills myself to fork and make such a change.

**Comment by itrrmp** [comment]
This would be a nice visual if there was the option to remove the background...
